demo/alloy-showcase-webapp/src/main/webapp/WEB-INF/component/alloy/button/menu/button.xhtml

```diff
@@ -56,7 +56,7 @@
 					<alloy:button includeViewParams="true">
 						<alloy:icon name="user" />
 						<alloy:outputText value="#{i18n['user']}" />
-						<f:param name="menuItem" value="view" />
```

Comment:
for consistency sake and better testing coverage, I altered the outputText value from view to user (commandMenu says you selected 'User' in feedback text after clicking on the User button).